openedx/features/edly/utils.py

```diff
@@ -333,3 +334,45 @@ def create_learner_link_with_permission_groups(user):
         user.groups.add(new_group)
 
     return user
+
+
+def get_current_site_invalid_certificate_context_data(default_html_certificate_configuration):
+    """
+    Gets current site's context data for invalid certificate.
+
+    Try to get current site's context data for invalid certificate from site configuration
+    or fallback to default html certificate configurations.
+
+    Arguments:
+        default_html_certificate_configuration (dict): Default html configurations dict.
+
+    Returns:
+        dict: Context data.
+    """
+    context = dict(default_html_certificate_configuration.get('default'))
+
+    current_site_configuration = get_current_site_configuration()
+
+    if not current_site_configuration:
+        return context
+
+    context['platform_name'] = current_site_configuration.get_value('platform_name', settings.PLATFORM_NAME)
+
+    branding = current_site_configuration.get_value('BRANDING', {})
+    context['logo_src'] = branding.get(
+        'logo',
+        default_html_certificate_configuration.get('default', {}).get('logo_src')
+    )
+
+    site_domain = current_site_configuration.get_value('site_domain')
+
+    default_logo_url = default_html_certificate_configuration.get('default', {}).get('logo_url')
+    context['logo_url'] = site_domain if site_domain else default_logo_url
+
+    default_company_privacy_url = default_html_certificate_configuration.get('default', {}).get('company_privacy_url')
+    context['company_privacy_url'] = '{}/privacy'.format(site_domain) if site_domain else default_company_privacy_url
```

Comment:
Our privacy pages are set on the WordPress and `site_domain` have lms base URL, I feel the URL will give 404

Reply:
Done